sdk/communication/communication-chat/CHANGELOG.md

```diff
@@ -1,11 +1,19 @@
 # Release History
 
-## 1.0.0-beta.4 (Unreleased)
+## 1.0.0-beta.4 (2021-02-08)
+
+- Updated `@azure/communication-chat` version.
 
 ### Breaking Changes
 
 - Replaced `CommunicationUser` with `CommunicationUserIdentifier`.
 - Replaced `CommunicationUserCredential` with `CommunicationTokenCredential`.
+- Support for CreateChatThreadResult and AddChatParticipantsResult to handle partial errors in batch calls.
```

Comment:
Are all these breaking changes? Doesn't look like that @juancamilor can you confirm?

Reply:
Yes, they are.

Reply:
The way they are phrased they sound they should be under ### Added.

Then under ### Breaking we can mention "Replaced X with Y", "Removed Z"

Reply:
@DominikMe could you give us a hand with the proper syntax ?